client/verta/verta/endpoint/update/rules.py

```diff
@@ -3,7 +3,7 @@
 import abc
 import json
 
-from ...external import six
+from verta.external import six
```

Comment:
Was there a functional reason for this change? The relative import `...external` is strongly preferred.

Reply:
I have no idea why this is changed 😱 
Need more tea

Reply:
Hmm I think that's pycharm copy-ing and pasting weird shennanigans. Let me fix all of them.

Reply:
Fixed.